smk/data_integration.smk

```diff
@@ -33,6 +33,14 @@ main_factor = None
 if config['MAIN_factor'] is not None:
     main_factor = config['MAIN_factor']
 
+plot_factor2 = None
+if config['PLOT_factor2'] is not None:
+    plot_factor2 = config['PLOT_factor2']
+
+plot_time = None
+if config['PLOT_time'] is not None:
+    plot_time = config['PLOT_time']
+
 # MIntO mode and database-mapping
```

Comment:
Could these config keys (and their equivalents in QC_2) be checked to be in config first, to avoid KeyError in snakemake for older config files?

Reply:
Good point. Fixed (406df3c)